Alignment/OfflineValidation/plugins/EopElecTreeWriter.cc

```diff
@@ -411,7 +411,7 @@ void EopElecTreeWriter::analyze(const edm::Event& iEvent, const edm::EventSetup&
 
     const unsigned int prescaleSize = hltConfig_.prescaleSize();
     for (unsigned int ps = 0; ps < prescaleSize; ps++) {
-      const unsigned int prescaleValue = hltConfig_.prescaleValue(ps, triggerName);
+      auto const prescaleValue = hltConfig_.prescaleValue<double>(ps, triggerName);
       if (prescaleValue != 1) {
```

Comment:
Is this ok with rounding effects?

Reply:
See https://github.com/cms-sw/cmssw/pull/39220#discussion_r975393631.